libraries/wallet/wallet.cpp

```diff
@@ -3012,6 +3133,67 @@ uint64_t wallet_api::get_asset_count()const
    return my->_remote_db->get_asset_count();
 }
 
+std::vector<unsigned char> string_to_vec(std::string incoming)
```

Comment:
fc already has that: https://github.com/bitshares/bitshares-fc/blob/master/src/variant.cpp#L664

Reply:
Well, almost. It uses char instead of uint8_t. I modified string_to_vec to use fc::from_hex. I will revisit this to see if there is a better way.